### PR TITLE
[BT-12061] Add phi 3.5 mini instruct example

### PR DESCRIPTION
**Pull request by derrick-yyang**
Used vLLM template to add Phi 3.5 mini-instruct model

Note: ignore the commit for using port `8080`. The default port `8000` is used

### CODE REVIEW

File: phi/phi-3.5-mini/README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+### OpenAI SDK (if you are using OpenAI compatible server)
+
+```
+from openai import OpenAI
+import os
+
+model_id = "a2345678" # Replace with your model ID
+
+client = OpenAI(
```
Comment (vshulman) [inline phi/phi-3.5-mini/README.md:69]: I think this OpenAI syntax actually changed slightly in recent weeks:
https://docs.baseten.co/api-reference/openai

File: phi/phi-3.5-mini/config.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  example_model_input: {"prompt": "what is the meaning of life"}
+  repo_id: microsoft/Phi-3.5-mini-instruct
+  openai_compatible: true
+  vllm_config:
+    tensor_parallel_size: 1
+    max_model_len: 10000
+requirements:
+  - vllm==0.5.4
+resources:
+  accelerator: A100
```
Comment (vshulman) [inline phi/phi-3.5-mini/config.yaml:13]: let's use a smaller GPU, since the beauty of a smaller model is that we can do that!

**Review by vshulman — CHANGES_REQUESTED** [review]
can we think of a way to reuse the `model.py` file without directly copying it?

File: phi/phi-3.5-mini/README.md (hunk 1 above)
Comment (derrick-yyang) [reply phi/phi-3.5-mini/README.md:69]: do you know the exact change? I don't see any difference between the api docs you sent me and the code in the readme

Comment (vshulman) [reply phi/phi-3.5-mini/README.md:69]: at a minimum, the `extra_body` field
might be worth testing this yourself to make sure this works

Comment (derrick-yyang) [reply phi/phi-3.5-mini/README.md:67]: updated both readmes 👍

**Review by vshulman — APPROVED** [review]
lgtm!

**Pull request merged**